docs/tutorials/quickstart-ohttp-relay.md

```diff
@@ -0,0 +1,92 @@
+# Quickstart ohttp relay
+
+We'll demostrate how you can setup an ohttp server on common cloud insfrastructure on something like [](483) using the `ohttp-relay` crate in Payjoin Dev Kit. This should take about 30 minutes. This tutorial assumes you have a basic understanding of the AWS infrastructure.
+
+The network parameters could be changed to support any Bitcoin network, but this tutorial will use the Bitcoin Signet network. We will ignore most errors and Payjoin Session persistence for simplicity. Reference the [`payjoin-cli` source](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli) reference implementation for more information on those implementation details.
+
+## Initial server setup
+
+We will be using a basic Ubuntu server in our tutorial. In this tutorial we will be running the server as a docker container behind nginx as a TLS proxy.
+
+First, you must install docker and nginx on your server.
+
+Install docker ond nginx on a fresh Ubuntu server:
+
+```sh
+sudo apt update && sudo apt upgrade -y # Ubuntu
+
+sudo apt install -y docker,io nginx
+
+sudo systemctl start docker nginx
+sudo systemctl enable docker nginx
+```
+
+## Deploy ohttp-relay with Docker
+
+We will want to build our docker image for our server to run as a container from.
+
+Clone our ohttp-relay from github and then build the docker image.
+
+```sh
+git clone https://github.com/payjoin/ohttp-relay.git
+
+cd ohttp-relay
+
+sudo docker build -t ohttp-relay .
+```
+
+## Configure Nginx as a Reverse Proxy
+
+Edit the existing `nginx.conf`.
+
+```sh 
+#/etc/nginx/nginx.conf
+load_modules /usr/lib/nginx/modules/ngx_mod_stream.so;
```

Comment:
I think we need instructions to get the nginx binary with ngx_mod_stream support. I don't believe it's compiled in by default.

Reply:
The simplest way to approach this would be I think to add `libnginx-mod-stream` to our install step